elements/pl-code/pl-code.py

```diff
@@ -184,7 +184,7 @@ def render(element_html, data):
 
     formatter_opts = {
         'style': 'friendly',
-        'cssclass': 'mb-2 rounded',
```

Comment:
This extra bottom margin was causing the code to not be vertically centered in the blocks. 

I couldn't think of any good reason for it to ever be on blocks of code so I just removed it always. Let me know if this is wrong.

Reply:
It looks like without it we get some weird behavior when we have two `<pl-code>` tags next to each other vertically:
![image](https://user-images.githubusercontent.com/1790491/168850046-42a39f74-e994-4dee-b73c-0cd2d4d07116.png)
(Hard to see, but those *are* supposed to be two separate blocks!)

We could add some hacky attribute to pl-code that turns that off, or it might be better to have an `inline` attribute that turns off margins/padding and renders inline instead of as a block?

Reply:
This is quite annoying. 

@mwest1066, @nicknytko  how about this: we can add some CSS with a fancy selector to override the margin of the pl-code element only in the case when it is the only child of one of the blocks?

Reply:
I think we can use selectors to fix this. @SethPoulsen Would this be for when it's the only pl-code item, or when it's the only child of its parent at all?

Reply:
Or we could just add contextual selectors for the specific case when it's pl-code inside of order blocks.

Reply:
@SethPoulsen I took the liberty of making a commit just now. Please check it out and revert if it's no good.

Reply:
Ya! That's _almost_ exactly what I was going to do, but I was going to make the selector a little more selective--if someone puts a `pl-code` inside a `pl-order-block` along with other content (and I have seen people do this), it should still have the margin. The margin should only be removed if it is the [only child](https://developer.mozilla.org/en-US/docs/Web/CSS/:only-child).

Reply:
Can you give an example of the heterogeneous mixture? I thought the `format="code"` applied to the entire order blocks element.

Reply:
OK, I think I see what you meant. I added some more rules for the mixed case as well as an ugly example to illustrate. (The example commit probably should be reverted, since it's ugly and redundant for end-users.)

Reply:
Sorry I wasn't more specific, I just meant to make the existing selector more specific. 

I guess you are right it doesn't matter much because we are only using that CSS class in the case when `format="code"`, but I still think its nice to have the selector be as specific as possible.